### ISSUE TITLE
`ensure in take_off': uninitialized constant Fastlane::VERSION

### ISSUE THREAD

**Issue by pesterhazy** [fastlane/fastlane#12301]
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I read [docs.fastlane.tools](https://docs.fastlane.tools)
- [x] I searched for [existing GitHub issues](https://github.com/fastlane/fastlane/issues)

### Issue Description

Running on Circle CI, we auto-update fastlane using `yes | gem update fastlane`.

Since a recent update (2.91), we run into a NameError.

##### Complete output when running fastlane, including the stack trace and command used
```
+ fastlane ios beta
[⠋] 🚀 [⠙] 🚀 [⠹] 🚀 [⠸] 🚀 [⠼] 🚀 /Users/distiller/project/.gem/ruby/2.0.0/gems/fastlane-2.91.0/fastlane/lib/fastlane/cli_tools_distributor.rb:110:in `ensure in take_off': uninitialized constant Fastlane::VERSION (NameError)
	from /Users/distiller/project/.gem/ruby/2.0.0/gems/fastlane-2.91.0/fastlane/lib/fastlane/cli_tools_distributor.rb:110:in `take_off'
	from /Users/distiller/project/.gem/ruby/2.0.0/gems/fastlane-2.91.0/bin/fastlane:20:in `<top (required)>'
	from /usr/local/bin/fastlane:23:in `load'
	from /usr/local/bin/fastlane:23:in `<main>'
Exited with code 1
```

### Environment

A macOS container on Circle CI using the `xcode: "8.3.3"` image.

Will fill in later

**Comment by SlaunchaMan** [comment]
I am also seeing this [in Xcode 7 images](https://travis-ci.org/AFNetworking/AFNetworking/jobs/366239284) in Travis.

**Comment by pesterhazy** [comment]
One data point: going back to (randomly chosen) 2.88 fixes things for me:

```
yes | gem install fastlane -v 2.88.0
```

**Comment by SlaunchaMan** [comment]
Yep, our 2.88 builds worked just fine.

**Comment by suryakant-t-yadav** [comment]
I am also going through the same issue. fastlane-2.91.0.
any help regarding this.

**Comment by rusty21** [comment]
Also seeing this issue on mac, v2.91.0

**Comment by chrisjingram** [comment]
I'm also seeing this on Mac OS (CircleCI), Fastlane v2.91.0

**Comment by cpk1** [comment]
Just to add some further details...
If I use RVM to switch to Ruby 2.4.1 which I don't use I can gem install fastlane and "fastlane --version" will work. However using the environment I typically work in - Ruby 2.0.0 using "bundle install --path custom/gem/path" I get:

[⠸] 🚀 bundler: failed to load command: fastlane (/custom/gem/path/ruby/2.0.0/bin/fastlane)
NameError: uninitialized constant Fastlane::VERSION

Followed by the same stack trace as above.

If I use my gemfile to use fastlane 2.89.0 everything works, however 2.90.0 and 2.91.0 are not working currently

**Comment by cpk1** [comment]
If I check out master and point my gemfile to use the local repo I don't get this error...

**Comment by cpk1** [comment]
Oops sorry for the confusion, I'm am getting this locally! Let me know if you want me to make another issue.
I have script that I use to start fastlane that does a "bundle install" and then "bundle exec fastlane".
Behavior I am seeing is if my gemfile has "gem 'fastlane'" I run into this, if I use "gemspec path: File.expand_path('/local/fastlane/master/checkout')" or "gem 'fastlane', '2.89.0'" I have no issues.

**Comment by joshdholtz** [comment]
@cpk1 I can't seem to think of anything that would be causing this 🤔 Can you try running `bundle update fastlane` instead when you `Gemfile` has `gem 'fastlane'` in it and see what happens?

**Comment by cpk1** [comment]
Doesn't seem to make a difference. 
Tried doing
bundle install --path  ~/github/Fastlane/vendor/bundle
bundle update fastlane
No clue what's going on but both locally and on jenkins we need to pin to 2.89.0, which is OK for now but I imagine at some point we are going to want to update :P.
I'll try to poke around some more 🤷‍♂️ 

Here's the top of bundle exec fastlane env if more info helps
### Stack

| Key                         | Value                                                                         |
| --------------------------- | ----------------------------------------------------------------------------- |
| OS                          | 10.12.6                                                                       |
| Ruby                        | 2.0.0                                                                         |
| Bundler?                    | true                                                                          |
| Git                         | git version 2.14.3 (Apple Git-98)                                             |
| Installation Source         | ~/github/Fastlane/vendor/bundle/ruby/2.0.0/bin/fastlane |
| Host                        | Mac OS X 10.12.6 (16G1314)                                                    |
| Ruby Lib Dir                | ~/.rvm/rubies/ruby-2.0.0-p648/lib                                             |
| OpenSSL Version             | OpenSSL 1.0.2n  7 Dec 2017                                                    |
| Is contained                | false                                                                         |
| Is homebrew                 | false                                                                         |
| Is installed via Fabric.app | false                                                                         |
| Xcode Path                  | /Applications/Xcode.app/Contents/Developer/                                   |
| Xcode Version               | 9.2                                                                           |

**Comment by cpk1** [comment]
Alright so after poking around some more it looks like the version of rubygems that is used to package the gem might be causing the issue? I am not familiar with how gems get packaged up and everything but I see that 2.90.0-2.92.1 (do not work for me) were all packaged with rubygems 2.4.5.5 whereas 2.89.0 (works for me) was packaged with 2.5.2.2 and 2.93.0.beta (works for me) was packaged with 2.5.2.1. All the gems that have "rubygems_version: 2.4.5.5" in their metadata file in the gem are giving me this error.

**Comment by joshdholtz** [comment]
@cpk1 That is a really interesting find 🤔 I will make the next release with `2.5.2.3` so see that helps you when version `2.93.0` gets build and pushed out later today/tomorrow

**Comment by cpk1** [comment]
@joshdholtz Todays release works! Thank you! Hopefully this fixes it for those folks on Circle CI as well.

**Comment by joshdholtz** [comment]
@cpk1 Glad to hear!

**Comment by joshdholtz** [comment]
@pesterhazy Does the latest release (`2.93.0`) solve this issue for you?

**Comment by Salierik** [comment]
@joshdholtz It works! Thank you very much!

**Comment by joshdholtz** [comment]
@cpk1 By the way, how were you able to view what version of ruby gems each version was packaged with? 🙃

**Comment by dominik-weber** [comment]
The issue seems to be resolved in 2.93.1 and 2.94.0. Thanks! 🙂
